app/main.py

```diff
@@ -1,3 +1,62 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: float) -> isinstance:
+        if isinstance(other, Distance):
+            res = other.km + self.km
+        else:
+            res = other + self.km
+        return Distance(res)
+
+    def __iadd__(self, other: int) -> isinstance:
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        else:
```

Comment:
else is redundant,  fix in all places

Reply:
Don`t understand how to work without Else

Reply:
When one condition is true, we return a value and do not proceed to the other block of code. You can just remove else or make this code with one  return